internal/server/v2/propertyvalues/golden/simple/containedIn1.json

```diff
@@ -24703,126 +24903,6 @@
               "dcid": "geoId/0625",
               "provenanceId": "dc/base/BaseGeos"
             },
-            {
-              "name": "Fort Bidwell",
-              "types": [
-                "City"
-              ],
-              "dcid": "geoId/0625030",
-              "provenanceId": "dc/base/BaseGeos"
-            },
```

Comment:
These seem suspicious?  Could it be from Census refresh?

Reply:
I took a closer look. This is not removed but moved to next page in containedIn2.json as we added more census tract. So this is good

Reply:
Turns out this is not a deletion but just a shift of these items since more census tracts are added.